devtools-ci-index.yaml

```diff
@@ -2193,154 +2193,198 @@
             ci_project: 'devtools'
             ci_cmd: 'cd ee_tests/ && /bin/bash cico_build_deploy.sh'
             timeout: '30m'
-        - 'devtools-test-end-to-end-ts-{test_url}-{test_suite}':
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}':
             test_url: openshift.io
-            test_suite: runTest
+            test_suite: smokeTest
+            cluster: cluster-one
             after: devtools-fabric8-test-build-master
             oso_token_creds: de0940f2-f31f-4186-85e2-37d42bddcf5b
             kc_refresh_token_creds: 807b706e-c54f-48d5-be93-56525cf499f0
             ee_test_start_time: '5 4-23/4 * * *'
             timeout: 60m
-        - 'devtools-test-end-to-end-ts-{test_url}-{test_suite}':
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}':
             test_url: openshift.io
-            test_suite: quickstartTest
+            test_suite: analyticTest
+            cluster: cluster-one
             oso_token_creds: de0940f2-f31f-4186-85e2-37d42bddcf5b
             kc_refresh_token_creds: 807b706e-c54f-48d5-be93-56525cf499f0
             ee_test_start_time: '5 5-23/4 * * *'
             timeout: 60m
             disabled: true
-        - 'devtools-test-end-to-end-ts-{test_url}-{test_suite}':
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}':
             test_url: openshift.io
-            test_suite: chequickstartTest
+            test_suite: cheTest
+            cluster: cluster-one
+            oso_token_creds: de0940f2-f31f-4186-85e2-37d42bddcf5b
+            kc_refresh_token_creds: 807b706e-c54f-48d5-be93-56525cf499f0
+            ee_test_start_time: '5 6-23/4 * * *'
+            timeout: 60m
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}':
+            test_url: openshift.io
+            test_suite: smokeTest
+            cluster: cluster-two
+            after: devtools-fabric8-test-build-master
+            oso_token_creds: e9a32e1e-0a38-4a96-896f-a266e1428cf9
+            kc_refresh_token_creds: 807b706e-c54f-48d5-be93-56525cf499f0
+            ee_test_start_time: '5 4-23/4 * * *'
+            timeout: 60m
+        - 'devtools-test-e2e-{test_url}-{test_suite}-{cluster}':
+            test_url: openshift.io
+            test_suite: analyticTest
+            cluster: cluster-two
+            oso_token_creds: de0940f2-f31f-4186-85e2-37d42bddcf5b
```

Comment:
This test is configured to run with the same user on both cluster 1 and 2 (lines 2214 and 2231), it doesn't look like a valid configuration.

Reply:
The critical variable is:  oso_token_creds - and this is configured differently for the different users. 

The "refresh" cred is not needed by the typescript tests. Once we remove all references to the old (javascript) tests, we can completely remove the refresh cred and variables from the yaml file.

Reply:
I'd like to remove the javascript tests in a separate pull request - as it will involve removing many lines of text in the yaml file.